extensions/parcel/src/api.ts

```
@@ -75,6 +83,30 @@ export function getApiKey(): string {
   return preferences.apiKey;
 }
 
+function getSystemLanguage(): string {
+  try {
+    const locale = new Intl.DateTimeFormat().resolvedOptions().locale;
+    const languageCode = locale.split("-")[0].toLowerCase();
+    if (languageCode && languageCode.length >= 2) {
+      return languageCode;
+    }
+  } catch {
```

Avoid empty catch blocks, always do something, if only logging a warning